watertap/unit_models/electrodialysis_1D.py

```diff
@@ -713,20 +721,22 @@ def _make_performance(self):
             self._pressure_drop_calculation()
 
             @self.Constraint(
+                self.flow_channel_set,
                 self.flowsheet().time,
-                self.diluate.length_domain,
-                doc="Express deltaP_term by the calculated pressure drop data, diluate.",
-            )
-            def eq_deltaP_diluate(self, t, x):
-                return self.diluate.deltaP[t, x] == -self.pressure_drop[t]
```

Comment:
Just so I understand correctly, does the existing model just assume `deltaP` (dP/dX) is constant, i.e., the same for each element? And now, with @hunterbarber 's modifications, we'd be calculating dP/dX per element rather than having a fixed dP/dx per element---is that correct? If not, please clarify.

Reply:
Effectively yes. In the old model, the limiting current density and pressure drop constraints were calculated using values at the inlet for variables such as velocity, friction factor, Sh, Re, etc. This indexes those variables for truer 1D predictions.